x-pack/plugins/security_solution/server/lib/detection_engine/rule_monitoring/logic/detection_engine_health/assets/install_assets_for_rule_monitoring.ts

```diff
@@ -79,20 +91,38 @@ const getAssetsForRuleMonitoring = (currentSpaceId: string) => {
   assetManagedTag.id = withSpaceId('fleet-managed');
   assetSecuritySolutionTag.id = withSpaceId('security-solution');
 
+  let tags: Array<SavedObjectsFindResult<TagAttributes>> | undefined;
+  try {
+    tags = await findTagsByName({
+      savedObjectsClient,
+      tagName: SECURITY_TAG_NAME,
+    });
+  } catch (err) {
+    // If no Security Solution tag exists, we will import it
+  }
+
+  const securityTagExists = tags && tags?.length > 0;
+
   // Update saved object references of the dashboard accordingly
   assetRuleMonitoringDashboard.references = assetRuleMonitoringDashboard.references.map(
     (reference) => {
       if (reference.id === 'fleet-managed-<spaceId>') {
         return { ...reference, id: assetManagedTag.id };
       }
       if (reference.id === 'security-solution-<spaceId>') {
-        return { ...reference, id: assetSecuritySolutionTag.id };
+        return {
+          ...reference,
+          id: tags?.[0]?.id ?? assetSecuritySolutionTag.id,
+        };
       }
 
       return reference;
     }
   );
-
+  if (securityTagExists) {
+    // If the Security Solution tag already exists, we will not import it
+    return [assetManagedTag, assetKibanaEventLogDataView, assetRuleMonitoringDashboard];
+  }
```

Comment:
And then, from `getAssetsForRuleMonitoring` we should never return the tag asset, and we should delete the `x-pack/plugins/security_solution/server/lib/detection_engine/rule_monitoring/logic/detection_engine_health/assets/tag_security_solution.json` file.